docs/references/astro/migrating-from-astro-community-sdk.mdx

```diff
@@ -1,5 +1,139 @@
 ---
+title: 'Migrating from the Astro community SDK'
 description: Learn how to migrate from the Astro community SDK to the Clerk Astro SDK.
 ---
 
 # Migrating from the Astro community SDK
+
+In June 2024, we introduced official support for Astro. This minimal guide highlights the breaking changes that were introduced and provides examples on how to resolve them.
+
+
+**Overview:**
+- Install `@clerk/astro`, uninstall `astro-clerk-auth`
+- Update import paths
+- Replace already deprecated APIs
+
+## Installation
+
+Uninstall the community SDK and install the new offical SDK for Astro.
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash filename="terminal"
+    npm uninstall astro-clerk-auth
+    npm install @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    yarn remove astro-clerk-auth
+    yarn add @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    pnpm remove astro-clerk-auth
+    pnpm add @clerk/astro
+    ```
+</CodeBlockTabs>
+
+## Update import paths
+We have kept breaking changes to the minimum and there is a great chance that it will not affect your existing codebase.
+As a quick solution find all the usages of `astro-clerk-auth/` and replace them `@clerk/astro/`
+
+```diff
+- import { clerkMiddleware } from "astro-clerk-auth/server"
++ import { clerkMiddleware } from "@clerk/astro/server"
+
+- import { SignIn } from "astro-clerk-auth/components/interactive"
++ import { SignIn } from "@clerk/astro/components/interactive"
+
+```
+
+
+## Breaking Changes
+
+### The integration
+
+When using the integration from `@clerk/astro` we are now hotloading clerk-js by default.
+
+```diff
+import { defineConfig } from "astro/config";
+import node from "@astrojs/node";
+- import clerk from "astro-clerk-auth/hotload";
++ import clerk from "@clerk/astro";
+
+export default defineConfig({
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
+
+**Bundled** \
+And in order to opt-in to the bundled clerk-js strategy you can do the following
+
+```diff
+import { defineConfig } from "astro/config";
+import node from "@astrojs/node";
+- import clerk from "astro-clerk-auth";
++ import clerk from "@clerk/astro/bundled";
+
+export default defineConfig({
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
+
+### Changes related to stores.
+
+Submodule `/stores` was replaced with `/client/stores`. For the full list of the stores see [here](TODO: add reference)
```

Comment:
Should we just create the doc with a title as placeholder, then we can update this link?

Reply:
Not sure what you mean here, do you want to drop the todo comment for now ?

Reply:
Sorry. I mean create the `/docs/references/astro/stores` page (or maybe just a section in the Overview depending on how much content there is). Then update the link to point to that page/section.

Reply:
Great, that was my intention, glad you are already working on this.